history/characters/1_azerothian.txt

```diff
@@ -17177,4 +17178,107 @@
 		add_trait=lunatic
 	}
 	608.3.10={ death=yes }
+}
+
+#dynasty=413
+5816={
+	name=Isiah			#Reginald's greatgrandfather
+	dna="0cebc000ca0"
```

Comment:
He doesn't have dynasty? Is it intended?

Reply:
nope, not intended, fixed, thx